css/css-color/t32-opacity-zorder-c-ref.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reference</title>
+<style>
+  table { border-spacing: 2px; }
+  td { border: 1px solid; }
+  td, div { width: 10px; height: 10px; }
+  div.opc { opacity: 0.99; }
+  div.green { background: green; }
+  div.up { margin-top: -10px; }
+  div.red { background: red; }
+</style>
+<body>
+  <p>Each of the following boxes should have a green or
+  very-nearly-green square inside of it:</p>
```

Comment:
Not super happy on 'very-nearly-green' but ok. 
Suggest adding explicit  "and no red"